lasagne/tests/layers/test_noise.py

```diff
@@ -138,7 +138,7 @@ def test_get_output_for_deterministic(self, layer):
         input = theano.shared(numpy.ones((3, 3)))
         result = layer.get_output_for(input, deterministic=True)
         result_eval = result.eval()
-        assert (result_eval == input.eval()).all()
+        assert (np.array(result_eval) == np.array(input.eval())).all()
```

Comment:
Thanks, yes that's what I meant. I will use floatX in the code and test it with the theano flag of floatX=float32 and floatX=falot64.